webpack.prod.js

```diff
@@ -29,20 +29,24 @@ module.exports = webpackMerge(commonConf, {
     new webpack.DefinePlugin({ SWITCH_DEBUG: '\'PRODUCTION\'' }),
     new ExtractTextPlugin('css/g4u.css'),
     new webpack.optimize.UglifyJsPlugin({
-      mangle: true,
+      mangle: {
+        screw_ie8: true
+      },
       sourceMap: false,
       compress: {
+        screw_ie8: true,
         dead_code: true, // jshint ignore:line
```

Comment:
Is this comment still necessary? If so, it may be necessary in the line before, too.